src/context/display/displayToplevel.ml

```diff
@@ -362,12 +370,20 @@ let collect ctx tk with_type =
 		let files = List.sort (fun (_,i1) (_,i2) -> -compare i1 i2) files in
 		List.iter (fun ((file,cfile),_) ->
 			let module_name = CompilationServer.get_module_name_of_cfile file cfile in
-			begin match List.rev cfile.c_package with
-				| [] -> ()
-				| s :: sl -> add_package (List.rev sl,s)
-			end;
-			Hashtbl.replace ctx.com.module_to_file (cfile.c_package,module_name) file;
-			process_decls cfile.c_package module_name cfile.c_decls
+			let dot_path = match cfile.c_package with
```

Comment:
`let dot_path = s_type_path (cfile.c_package,module_name)` in

Reply:
I knew I was missing something... I was loooking for `s_path`. :)